server/provisioning/docker/spark-worker/files/default_cmd

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+source /root/spark_files/configure_spark.sh
+
+IP=$(ip -o -4 addr list eth0 | perl -n -e 'if (m{inet\s([\d\.]+)\/\d+\s}xms) { print $1 }')
```

Comment:
Hmm, whilst this may work, I'm uncomfortable with this type of IP address discovery. Surely, IP addresses to use should be injected in via the DevOps provisioning layer (e.g. as an environment variable)?

My primary concern here is that the docker container may have multiple NICs and eth0 may not be our "external" interface.